.circleci/config.yml

```diff
@@ -384,7 +384,6 @@ jobs:
       - run: make docker-child_chain CHILD_CHAIN_IMAGE_NAME="omisego/child_chain"
       - run: make docker-watcher WATCHER_IMAGE_NAME="omisego/watcher"
       - run: make docker-watcher_informational WATCHER_INFORMATIONAL_IMAGE_NAME="omisego/watcher_informational"
-      - run: docker-compose -f docker-compose.yml -f priv/docker-compose.geth-ganache.yml pull
```

Comment:
@InoMurko Can I remove this line? The pull is discarding the images from the `make docker-*` commands above it.

Reply:
oh shit, yeah, remove if it works or pull what needs to be pulled geth, plasma-contracts and postgres

Reply:
does it really override `make docker-*`? I would have though it doesn't pull if it exists in the local repo (where make docker-* pushes it).

Reply:
I couldn't find the official reference for this behaviour, but the logs at https://circleci.com/gh/omisego/elixir-omg/27723 can confirm it. It's pulling the images down again.

And compose still works after removing this line because it automatically fetches missing images, i.e. `trufflesuite/ganache-cli:latest`, as confirmed in https://circleci.com/gh/omisego/elixir-omg/27775.